cmd/setup_darwin.go

```diff
@@ -17,13 +17,146 @@
 package cmd
 
 import (
+	"bytes"
 	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/spf13/viper"
+
+	"github.com/spf13/cobra"
 )
 
-func PrintSetup() {
-	fmt.Println("Please run the following commands to setup routing for the meta-data service:")
-	fmt.Println("")
-	fmt.Println("\tsudo ifconfig lo0 169.254.169.254 alias")
-	fmt.Println("\techo \"rdr pass on lo0 inet proto tcp from any to 169.254.169.254 port 80 -> 127.0.0.1 port 9091\" | sudo pfctl -ef -")
-	fmt.Println("")
+var (
+	setupShortHelp = "Print setup information"
+	setupLongHelp  = `By default, this command will print a script that can be used to set up IMDS routing.
+If you trust us enough, you can run sudo weep setup --write.
```

Comment:
I think should be `you can run sudo weep setup --commit`?

Reply:
Yup, good catch